tests/debugger/test_debugger_symdb.py

```diff
@@ -33,6 +33,33 @@ def _assert_symbols_uploaded(self):
                 )
 
         assert not errors, "Found system-tests-errors:\n" + "\n".join(f"- {err}" for err in errors)
+        self._assert_debugger_controller_exists()
+
+    def _assert_debugger_controller_exists(self):
+        pattern = r"[Dd]ebugger[_]?[Cc]ontroller"
```

Comment:
If there is only one char in a char set, you can directly use the cahr

```suggestion
        pattern = r"[Dd]ebugger_?[Cc]ontroller"
```

Reply:
Underscore is optional. In python it's called debugger_controller, in java and dotnet - DebuggerController

Reply:
yep, the `?` just after the `_` will do the trick : https://regex101.com/r/J1P4r6/1